common/src/main/java/org/apache/drill/common/collections/MapWithOrdinal.java

```diff
@@ -54,7 +53,7 @@
 public class MapWithOrdinal<K, V> implements Map<K, V> {
   private final static Logger logger = LoggerFactory.getLogger(MapWithOrdinal.class);
 
-  private final Map<K, Entry<Integer, V>> primary = Maps.newLinkedHashMap();
+  private final Map<K, Entry<Integer, V>> primary = new LinkedHashMap<>();
   private final IntObjectHashMap<V> secondary = new IntObjectHashMap<>();
 
   private final Map<K, V> delegate = new Map<K, V>() {
```

Comment:
What is the purpose of the `delegate`? I'd recommend limiting this class changes to updated imports only and refactor it in a separate JIRA/PR.

Reply:
OK, I have removed all these changes and will open a separate PR with them. So now this PR contains only changes required for compiling the code.